React/Views/RCTActivityIndicatorView.m

```diff
@@ -66,33 +69,32 @@ - (void)setActivityIndicatorViewStyle:(UIActivityIndicatorViewStyle)activityIndi
   }
 }
 
-- (void)setColor: (RCTUIColor*)color
+- (void)setColor:(RCTUIColor*)color
 {
   if (_color != color) {
+    if (color != nil) {
+      CGFloat r, g, b, a;
+      [[color colorUsingColorSpaceName:NSCalibratedRGBColorSpace] getRed:&r green:&g blue:&b alpha:&a];
```

Comment:
Isn't the color we get back not trustable anymore since it moved from updateLayer to setColor: which may not be part of the draw callback? See "Update Custom Views using Specific Methods" at https://developer.apple.com/documentation/xcode/supporting_dark_mode_in_your_interface

Reply:
You're right. I just thought I was being smart by not creating the filter every frame :smile: